meinberlin/apps/plans/exports.py

```diff
@@ -22,7 +22,8 @@ class DashboardPlanExportView(a4dashboard_mixins.DashboardBaseMixin,
     permission_required = 'meinberlin_plans.export_plan'
     model = models.Plan
     fields = ['title', 'description', 'contact', 'district', 'topics',
-              'cost', 'duration', 'status', 'participation', 'organisation']
+              'cost', 'duration', 'status', 'participation', 'organisation',
+              'created', 'modified']
```

Comment:
I think, we do not need the fields in here, because we do add them to the virtual fields in the `get_virtual_fields`. Also, if we only add them to the virtual fields later, it should be added to the end, which was a NTH (nice to have).